docker-compose.yml

```diff
@@ -17,7 +17,8 @@ services:
     volumes:
       - data:/data
       - ./tools/aplus-init.sh:/srv/aplus-init.sh:ro
-      - ./tools/test-bench-course-mod.py:/srv/test-bench-course-mod.py:ro
+      - ./tools/simple_course.py:/srv/simple_course.py:ro
```

Comment:
as far as I can read this "overrides" (covers the data sets) that were earlier created by Mankro's script :question: